content/docs/dvclive/get-started.md

```diff
@@ -105,81 +99,108 @@ dvclive
 
 <admon type="info" icon="book">
 
-You can find more details about the structure of files generated by DVCLive in
+You can find more details about the structure of files generated by DVCLive on
 the [How it Works](/doc/dvclive/how-it-works) page.
 
 </admon>
 
-## DVC Integration
-
-To build on the previous section, you can optionally wrap our code with DVC to
-run [DVC experiments](/doc/user-guide/experiment-management/) and enable new
-ways to compare and visualize our experiments.
-
-You can find below an example `dvc.yaml` containing a single stage `train`, that
-runs the script from previous steps and tracks the
-[DVCLive outputs](/doc/dvclive/how-it-works) as `dvc metrics` and `dvc plots`:
-
-```yaml
-stages:
-  train:
-    cmd: python train.py
-    deps:
-      - train.py
-    metrics:
-      - dvclive/metrics.json:
-          cache: false
-    plots:
-      - dvclive/plots:
-          cache: false
-```
+### Track the results
 
-### Run DVC Experiments
+Unless you are:
 
-You can now
-[run experiments](/doc/user-guide/experiment-management/running-experiments)
-with `dvc exp run`:
+- A)
+  [Running DVC experiments](/doc/user-guide/experiment-management/running-experiments)
+  and DVCLive is used inside a stage of `dvc.yaml`.
+- B) Passed [`save_dvc_exp=True`](/doc/dvclive/api-reference/live#parameters) to
+  [`Live()`](/doc/dvclive/api-reference/live).
 
-```cli
-$ dvc exp run
-Running stage 'train':
-> python train.py
-...
-```
+Upon each execution, DVCLive will **overwrite** the results in the workspace.
+
+You would have to use one of the options above or take care of tracking the
+results with `git` between executions.
```

Comment:
Not happy about this section at all, but not sure how to simplify

Reply:
Can we include/recommend `save_dvc_exp=True` in "Set up DVCLive"? At this point, it automatically turns off for any situation where it could cause problems, so it seems better to just recommend it at the start.

Then you could start this section by explaining that dvclive expects results to be tracked by git and dvc, so it will override results each time (take out the "unless" part -- it always overrides results except if `resume=True`, right?). If you haven't already set up a dvc pipeline, `dvc_exp_save` takes care of saving the results for you as dvc experiments so you don't have to manually commit each run.

What do you think?

Reply:
Makes sense